systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -106,11 +111,36 @@
 @Tag(CONNECT)
 @Tag(CONNECT_COMPONENTS)
 @SuppressWarnings({"checkstyle:ClassDataAbstractionCoupling"})
+@SuiteDoc(
+    description = @Desc("Verifies the deployment, manual rolling update, and undeployment of Kafka Connect components."),
+    beforeTestSteps = {
+        @Step(value = "Deploy uber operator across all namespaces, with custom configuration", expected = "Uber operator is deployed"),
+        @Step(value = "Deploy management Pod for accessing all other Pods", expected = "Management Pod is deployed")
+    },
+    labels = {
+        @Label(value = "connect")
+    }
+)
 class ConnectST extends AbstractST {
 
     private static final Logger LOGGER = LogManager.getLogger(ConnectST.class);
 
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Verifies the deployment, manual rolling update, and undeployment of Kafka Connect components."),
+        steps = {
+            @Step(value = "Initialize Test Storage", expected = "Test storage instance is created with required context"),
+            @Step(value = "Define expected configurations", expected = "Configurations are loaded from properties file"),
+            @Step(value = "Create and wait for resources", expected = "Kafka resources, including NodePools and KafkaConnect instances, are created and become ready"),
+            @Step(value = "Annotate for manual rolling update", expected = "KafkaConnect components are annotated for a manual rolling update"),
+            @Step(value = "Perform and wait for rolling update", expected = "KafkaConnect components roll and new pods are deployed"),
+            @Step(value = "Verify Kafka Connect pod", expected = "Pod configurations and annotations are verified"),
```

Comment:
Here as well - what does verify mean? I see it in the expected part, but I would mention it in the `value`